virtualization/Docker/setup_docker_prereqs

```diff
@@ -62,6 +63,10 @@ if [ "$INSTALL_SSOCR" == "yes" ]; then
   virtualization/Docker/scripts/ssocr
 fi
 
+if [ "$INSTALL_DLIB" == "yes" ]; then
+  pip3 install --no-cache-dir face_recognition==1.0.0
```

Comment:
Just wondering, is there a way to just install dlib? I assume that is a dependency of face_recognition?

Reply:
Yeah, the `dlib` PyPi package automatically builds it, which is a dependency of `face_recognition`. I could just only install `dlib`, but `face_recognition` isn’t a very big package to make a difference.

Reply:
The I prefer to just install dlib. The requirements will take sure of installing dlib.

Especially since it can get confusing when face_recognition gets updated and this script install an older version.